src/features/world/scenes/PlazaScene.ts

```diff
@@ -66,11 +66,16 @@ export const PLAZA_BUMPKINS: NPCBumpkin[] = [
     npc: "grimtooth",
     direction: "left",
   },
-  // {
-  //   x: 120,
-  //   y: 170,
-  //   npc: "gabi",
-  // },
+  {
+    x: 120,
+    y: 170,
+    npc: "gabi",
+  },
+  {
```

Comment:
Glinteye is showing even when the flag is turned off.

Consider first checking whether the flag is on before including in the list of bumpkins 👍 

E.g.

```
    let bumpkins = PLAZA_BUMPKINS;
    
    if (flagOn) {
     // Add glinteye
     }
    this.initialiseNPCs(bumpkins);
```